app/assets/stylesheets/pages/home/_banner.scss

```diff
@@ -27,6 +33,26 @@
 		font-size: 36px;
 		margin-top: 0;
 		margin-bottom: 25px;
+
+		@media only screen and (max-width: 1024px){
```

Comment:
you don't have to change the font this much, just change it when it's hit 480px width